Makefile

```diff
@@ -0,0 +1,60 @@
+
+# Disable make's implicit rules, which are not useful for golang, and slow down the build
+# considerably.
+.SUFFIXES:
+
+all: bin/confd bin/confd.static
```

Comment:
That's a good question.  No idea.  I meant to follow up to see if you knew why we had both.

I'd prefer to just have the static build and get rid of the .static suffix.